lib/funky/facebook_graph_root_node.rb

```diff
@@ -0,0 +1,47 @@
+module Funky
+  class FacebookGraphRootNode
```

Comment:
@bruce-fs Much better! I would remove `Facebook` since this whole library is about Facebook.